diff -ruN gl4es/CMakeLists.txt gl4es_patched/CMakeLists.txt
--- gl4es/CMakeLists.txt	2024-09-08 00:30:12.301721000 +0200
+++ gl4es_patched/CMakeLists.txt	2024-08-27 23:23:38.818684000 +0200
@@ -2,11 +2,15 @@
 
 project(gl4es LANGUAGES C)
 
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/bin)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib)
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib)
+set(USE_ANDROID_LOG ON)
+set(ANDROID ON)
+#set(EGL_WRAPPER ON)
+
+#set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/bin)
+#set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib)
+#set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/lib)
 
-link_directories(${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+#link_directories(${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 
 option(PANDORA "Set to ON if targeting an OpenPandora device" ${PANDORA})
 option(PYRA "Set to ON if targeting an Dragonbox Pyra device" ${PYRA})
@@ -226,7 +230,7 @@
     if (${ARGC} EQUAL 5)
         add_test(${test_name}
             ${CMAKE_COMMAND}
-            -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+            #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
             -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
             -D TEST_FILENAME=${test_filename}
             -D CALLS=${calls_count}
@@ -237,7 +241,7 @@
         if (${ARGV4} STREQUAL "NOEXTRACT_RANGE")
             add_test(${test_name}
                 ${CMAKE_COMMAND}
-                -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+                #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
                 -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
                 -D TEST_FILENAME=${test_filename}
                 -D CALLS=${calls_count}
@@ -247,7 +251,7 @@
         else (${ARGV4} STREQUAL "NOEXTRACT_RANGE")
             add_test(${test_name}
                 ${CMAKE_COMMAND}
-                -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+                #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
                 -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
                 -D TEST_FILENAME=${test_filename}
                 -D CALLS=${calls_count}
@@ -259,7 +263,7 @@
     else (${ARGC} EQUAL 5)
         add_test(${test_name}
             ${CMAKE_COMMAND}
-            -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+            #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
             -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
             -D TEST_FILENAME=${test_filename}
             -D CALLS=${calls_count}
@@ -272,7 +276,7 @@
     if (${ARGC} EQUAL 5)
         add_test(${test_name}_GLES${GLES}
             ${CMAKE_COMMAND}
-            -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+            #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
             -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
             -D TEST_FILENAME=${test_filename}
             -D CALLS=${calls_count}
@@ -283,7 +287,7 @@
     else (${ARGC} EQUAL 5)
         add_test(${test_name}_GLES${GLES}
             ${CMAKE_COMMAND}
-            -D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+            #-D LIBRARY_FOLDER=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
             -D TESTS_DIRECTORY=${CMAKE_SOURCE_DIR}/tests
             -D TEST_FILENAME=${test_filename}
             -D CALLS=${calls_count}
Datei gl4es/.git ist ein normale Datei, während Datei gl4es_patched/.git ein Verzeichnis ist.
diff -ruN gl4es/src/CMakeLists.txt gl4es_patched/src/CMakeLists.txt
--- gl4es/src/CMakeLists.txt	2024-09-08 00:30:12.317722000 +0200
+++ gl4es_patched/src/CMakeLists.txt	2024-08-27 23:00:19.877168000 +0200
@@ -221,7 +221,7 @@
     endif()
     
     if(CMAKE_SHARED_LIBRARY_SUFFIX MATCHES ".so")
-        set_target_properties(GL PROPERTIES SUFFIX ".so.1")
+        set_target_properties(GL PROPERTIES SUFFIX ".so") #.1
     endif()
     install(TARGETS GL
       LIBRARY
@@ -249,7 +249,7 @@
         target_link_libraries(EGL ${log-lib})
     endif()
     if(CMAKE_SHARED_LIBRARY_SUFFIX MATCHES ".so")
-        set_target_properties(EGL PROPERTIES SUFFIX ".so.1")
+        set_target_properties(EGL PROPERTIES SUFFIX ".so") #.1
     endif()
 endif()
